tools/deploy.mjs

```diff
@@ -4,7 +4,7 @@ process.exit(
   spawnSync(
     'node',
     [
-      './node_modules/@webosose/ares-cli/bin/ares-install.js',
+      './node_modules/@webos-tools/cli/bin/ares-install.js',
```

Comment:
Should we really be reaching into `node_modules` like that? It seems likely to break in exactly this way. I would prefer [libnpmexec](https://github.com/npm/cli/tree/latest/workspaces/libnpmexec) or at least `./node_modules/.bin/ares-install`.

Reply:
If you're fine with `shell: true`, the latter is easily done.

Reply:
Is that necessary for Windows? (Does it still use `.cmd` files on Windows?) On Linux, just replacing `node` with `./node_modules/.bin/ares-install` and removing the script path from the args seems to work.

Reply:
It's necessary on Windows. To quote Node docs:

> On Windows, however, .bat and .cmd files are not executable on their own without a terminal, and therefore cannot be launched using child_process.execFile().

https://nodejs.org/api/child_process.html#spawning-bat-and-cmd-files-on-windows

Reply:
Then it's fine. Does everything else work on Windows?